test/framework/modules/imkl/10.3.12.361

```diff
@@ -13,7 +13,7 @@ module-whatis {Intel Math Kernel Library is a library of highly optimized,
  applications that require maximum performance. Core math functions include
  BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more. - Homepage: http://software.intel.com/en-us/intel-mkl/}
 
-set root	/var/folders/6y/x4gmwgjn5qz63b7ftg4j_40m0000gn/T/tmpZg_7A8
+set root	/tmp/eb-bI0pBy/eb-DmuEpJ/eb-leoYDw/eb-UtJJqp/tmp8P3FOY
```

Comment:
I need to get this fixed, this shouldn't happen (it's a problem of a test that fiddles with this file in place failing, and then not restoring this file to what it was)